shell/gpu/gpu_surface_gl_skia.cc

```diff
@@ -157,10 +156,12 @@ static sk_sp<SkSurface> WrapOnscreenSurface(GrDirectContext* context,
   );
 }
 
-bool GPUSurfaceGLSkia::CreateOrUpdateSurfaces(const SkISize& size) {
-  if (onscreen_surface_ != nullptr &&
-      size == SkISize::Make(onscreen_surface_->width(),
-                            onscreen_surface_->height())) {
+bool GPUSurfaceGLSkia::CreateOrUpdateSurfaces(
+    std::unique_ptr<ViewUnit>& view_unit,
```

Comment:
Probably should be const or just a raw pointer.

If it can't be const consider making it a shared pointer.